src/platform/platform_linux.cpp

```diff
@@ -159,11 +159,6 @@ bool mp::platform::is_alias_supported(const std::string& alias, const std::strin
 
 bool mp::platform::is_remote_supported(const std::string& remote)
 {
-    const auto& driver = utils::get_driver_str();
-    // TODO: Remove this constraint once we support other image remotes for the LXD backend
-    if (driver == QStringLiteral("lxd") && (remote != "release" && remote != "daily"))
-        return false;
-
```

Comment:
We still can't support `appliance:`, right?

Reply:
Right, but the `appliance` remote is streams based, so it's smart enough to figure out there are no LXD images available, so they won't show up in `multipass find` or be available for `launch`.  So once those images are there, it should just start working without any code intervention.

Reply:
D'uh, I implemented that myself ;P